pages/filterguide.tsx

```diff
@@ -0,0 +1,224 @@
+import Head from 'next/head';
+import React from 'react';
+import Link from 'next/link';
+import { ListGroup, Container, Accordion, Card, Button } from 'react-bootstrap';
+
+const { Item } = ListGroup;
+
+export default function FilterGuide(): JSX.Element {
+  return (
+    <React.Fragment>
+      <Head>
+        <title>Filter Guide</title>
+      </Head>
+      <Container fluid className="ml-2">
+        <h1>ID Tool Filter Guide</h1>
+        <br/>
+        <Accordion>
+          <Card>
+            <Card.Header>
+              <Accordion.Toggle as={Button} variant="light" eventKey="0">
+                Location
+              </Accordion.Toggle>
+            </Card.Header>
+            <Accordion.Collapse eventKey="0">
+              <ListGroup>
+                <Item key="at leaf vein angles">
```

Comment:
We should sort these alpha I think.

Reply:
Or group them by general plant area? Leaves, stems, roots? @Megachile

Reply:
Either way seems fine to me.